### ISSUE TITLE
How to disable baritone saving chunks when the game is closed?

### ISSUE THREAD

**Issue by bebeli555** [cabaletta/baritone#1977]
So when u close the game baritone starts loading all the chunks around you which causes the game to freeze for a whole minute before it finally closes. How do i disable this? it happens even if u never had baritone used in that game session

**Comment by Tragafantas** [comment]
I don't have that problem

**Comment by dominikaaaa** [comment]
> So when u close the game baritone starts loading all the chunks around you

That's not possible.

**Comment by UltraBlackLinux** [comment]
Evidence: #show-us-the-log

**Comment by dominikaaaa** [comment]
Pro coder bebeli555 doesn't need log files, just look at his amazing code 

https://github.com/bebeli555/ElytraBot/blob/31c995bcbf4affcac4cd677c6f758331c6b1cc4e/src/main/java/me/bebeli555/ElytraBot/Main.java#L25-L58

**Comment by bebeli555** [comment]
Yeah it isnt baritone nvm. Seems to be future causing this. @dominikaaaa Thanks. ive learned quite alot after i wrote that code. Gonna redo that class sometime if neccessary.
